src/components/features/ProductSearch/ProductSearch.js

```diff
@@ -21,7 +21,7 @@ const ProductSearch = () => (
             </li>
             <li>
               <a href='#' value=''>
-                FURNITURE
+                Furniture
```

Comment:
w tym pliku nie ma działających linków. To nie jest częścią zadania?

Reply:
Linki są zdefioniowane w MenuBar i wydaje mi się, że tego dotyczy zadanie